crypt.go

```diff
@@ -675,6 +675,52 @@ func setLUKS2KeyslotPreferred(devicePath string, slot int) error {
 	return nil
 }
 
+// InitializeLUKS2ContainerOptions carries options for initializing LUKS2
+// containers.
+type InitializeLUKS2ContainerOptions struct {
+	// MetadataSize sets the size of the LUKS2 metadata (JSON) area.The
+	// value includes 4096 bytes for the binary metadata. According to LUKS2
+	// specification and cryptsetup(8), only these values are valid: 16, 32,
+	// 64, 128, 256, 512, 1024, 2048 and 4096 kB.
+	MetadataSize int
```

Comment:
I wonder given the constraints on both sizes whether to just let speficy a number of KiB, and call both options then:

MetadataKiBSize  and KeyslotsAreaKiBSize